src/frontend/komponenter/Fagsak/Brevmottaker/LeggTilEndreBrevmottakerModal.tsx

```diff
@@ -109,122 +117,133 @@ export const LeggTilEndreBrevmottakerModal: React.FC = () => {
-                                    skjema.felter.organisasjonsnummer.validerOgSettFelt(
+                                    skjema.felter.fødselsnummer.validerOgSettFelt(
                                         event.target.value
                                     );
                                 }}
                             />
-                            <FamilieInput
-                                {...skjema.felter.navn.hentNavBaseSkjemaProps(
-                                    skjema.visFeilmeldinger
-                                )}
-                                label={'Kontaktperson i organisasjonen'}
-                                description={
-                                    'Navnet vises etter organisasjonsnavnet slik “Organisasjon AS v/ Navn Navnesen”'
-                                }
-                                onChange={(event): void => {
-                                    skjema.felter.navn.validerOgSettFelt(event.target.value);
-                                }}
-                            />
-                        </>
-                    )}
-                </StyledFieldset>
+                        )}
+                        {adresseKilde === AdresseKilde.OPPSLAG_ORGANISASJONSREGISTER && (
+                            <>
+                                <FamilieInput
+                                    {...skjema.felter.organisasjonsnummer.hentNavBaseSkjemaProps(
+                                        skjema.visFeilmeldinger
+                                    )}
+                                    label={'Organisasjonsnummer'}
+                                    onChange={(event): void => {
+                                        skjema.felter.organisasjonsnummer.validerOgSettFelt(
+                                            event.target.value
+                                        );
+                                    }}
+                                />
+                                <FamilieInput
+                                    {...skjema.felter.navn.hentNavBaseSkjemaProps(
+                                        skjema.visFeilmeldinger
+                                    )}
+                                    label={'Kontaktperson i organisasjonen'}
+                                    description={
+                                        'Navnet vises etter organisasjonsnavnet slik “Organisasjon AS v/ Navn Navnesen”'
+                                    }
+                                    onChange={(event): void => {
+                                        skjema.felter.navn.validerOgSettFelt(event.target.value);
+                                    }}
+                                />
+                            </>
+                        )}
+                    </StyledFieldset>
+                </FlexContainer>
+            </StyledModalBody>
+            <Modal.Footer>
                 <ModalKnapperad>
```

Comment:
Her må vi fjerne ModalKnapperad og de tomme <> for at Footer skal bli stylet sånn som Aksel foreslår 😊

Reply:
👍🏻